child_compassion/model/project_compassion.py

```diff
@@ -127,8 +144,9 @@ def update_informations(self, cr, uid, ids, context=None):
             them accordingly. """
         if not isinstance(ids, list):
             ids = [ids]
-
         for project in self.browse(cr, uid, ids, context):
+            # mock variable
+            values = self._get_age_groups(cr, uid, project, context)
```

Comment:
`_get_age_groups` returns a list, and in the next line you override the `values` variable with a dictionary returned by the `_update_program_info` method. So the return value of `get_age_groups` is not useful, don't capture it.